assets/src/scripts/components/hydrograph/layout.js

```diff
@@ -7,7 +7,7 @@ const ASPECT_RATIO_PERCENT = `${100 * ASPECT_RATIO}%`;
 const MARGIN = {
     top: 20,
     right: 75,
-    bottom: 50,
+    bottom: 70,
```

Comment:
There will be be some more fanciness around for the legend to be responsive, but there's already a separate task for that ironing out issues on the legend.